### DISCUSSION TITLE
Restoring Kixeye Updates in Future Backyard Monsters?

### DISCUSSION THREAD

**Discussion by BenHeroPH** [bym-refitted/backyard-monsters-refitted#17] [Q&A] [answered]
The moment I found out from my friend that my childhood game, 'Backyard Monsters,' can be played through this GitHub, I was excited to experience it again and feel the nostalgia. Now, my question is, **is it possible to bring back all the updates that were made by Kixeye in the future?**

**Comment by React1-X** [comment, marked as answer]
> The moment I found out from my friend that my childhood game, 'Backyard Monsters,' can be played through this GitHub, I was excited to experience it again and feel the nostalgia. Now, my question is, **is it possible to bring back all the updates that were made by Kixeye in the future?**

Answer: Yes, the primary goal of this project is to preserve a 1:1 restoration of the original game, therefore, we will be including almost all updates from the original.
